amr-wind/turbulence/LES/OneEqKsgs.cpp

```diff
@@ -41,7 +41,7 @@ OneEqKsgsM84<Transport>::OneEqKsgsM84(CFDSim& sim)
     }
 
     {
-        const std::string coeffs_dict = this->model_name() + "_coeffs";
+        const std::string coeffs_dict = model_name() + "_coeffs";
```

Comment:
I doubt this will fix the issue... as the method is still declared virtual.

Reply:
darn, I need to set up clang-tidy on my machine instead of depending on CI

Reply:
You can use it on eagle with `module load llvm` or rhodes when eagle is down.